packages/firebase_analytics/firebase_analytics/example/lib/main.dart

```diff
@@ -67,18 +67,18 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   Future<void> _sendAnalyticsEvent() async {
+    EventParameters params = EventParameters();
+
+    params.addString('string', 'string');
+    params.addNumber('int', 42);
+    params.addNumber('long', 12345678910);
+    params.addNumber('double', 42.0);
+    params.addString('bool', true.toString());
+    // Only strings and numbers (ints & doubles) are supported for GA custom event parameters:
+    // https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview
     await widget.analytics.logEvent(
       name: 'test_event',
-      parameters: <String, dynamic>{
-        'string': 'string',
-        'int': 42,
-        'long': 12345678910,
-        'double': 42.0,
-        // Only strings and numbers (ints & doubles) are supported for GA custom event parameters:
-        // https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview
-        'bool': true.toString(),
```

Comment:
WDYT about introduce an `addBool` method, which behind the scenes saves it as `value.toString()`?

Reply:
I would prefer not as it isn't stored as a boolean and hence misleading, `true.toString()` is painless enough.